docker/data-generator/data-generator.py

```diff
@@ -15,34 +15,49 @@
 from datetime import datetime
 from time import sleep
 
-import redis
+from pymongo import MongoClient
+from pymongo.collection import Collection
 from pytz import timezone
 from sklearn.datasets import load_wine
 
-REDIS_CLINET = redis.Redis(host="redis", port=6379, db=0)
-KR_TZ = timezone("Asia/Seoul")
 
+def create_collection(mongo_client: MongoClient) -> Collection:
+    """Create collection."""
+    database = mongo_client["mongo"]
+    if "wine_data" not in database.list_collection_names():
+        database.create_collection("wine_data")
+    return database["wine_data"]
```

Comment:
명시적으로 collection을 만들어주는 코드 좋습니다 :)
그런데 `pymongo` Client를 key로 접근하게 되면 자동으로 DB -> Collection 순서로 mapping 해주는 기능이 있어 함께 공유드립니다.

```suggestion
def create_collection(mongo_client: MongoClient) -> Collection:
    """Create collection."""
    return mongo_client["mongo"]["wine_data"]
```

Reply:
- [Official Tutorial docs: database, collection creation](https://pymongo.readthedocs.io/en/stable/tutorial.html#getting-a-database)

### **Key Phrase**
 _An important note about collections (and databases) in MongoDB is that they are created lazily - none of the above commands have actually performed any operations on the MongoDB server. **Collections and databases are created when the first document is inserted into them.**_

Reply:
수정하였습니다~